.gitignore

```diff
@@ -13,6 +13,7 @@ graal_dumps/
 target/
 *.class
 *.log
+lib/scala/ast/src/main/scala/org/enso/ast/Ast.scala
```

Comment:
This is going into the wrong place. It should go into a `target` folder like all generated code.